www/_data/tools.yml

```diff
@@ -1,9 +1,10 @@
-# -   name: Example Tool
-#     image: cordova.png
-#     url: http://example.com
+# -   name: GapDebug
+#     image: gapdebug-logo100.png
+#     url: https://www.genuitec.com/products/gapdebug/
 #     description: >
-#         A description that can
-#         span multiple lines.
+#         GapDebug is a free tool for debugging and managing hybrid-mobile Android and iOS apps. 
+#         With Drag and drop App Installation, Instant debugging, and full cross-platform support, 
+#         GapDebug changes how you view debugging for good.
```

Comment:
Could you please add your tool as the last item in the list instead? Sorry for the inconvenience! You may just copy the example to the bottom of the file, change its data to reflect yours, and then resubmit the PR. That way we won't have to change your commit very much before merging.

Thank you!